android/fakedapp/src/main/res/values/strings.xml

```diff
@@ -20,7 +20,8 @@
     <string name="label_sign_txn_x1">Sign txn x1</string>
     <string name="label_sign_txn_x3">x3</string>
     <string name="label_sign_txn_x20">x20</string>
-    <string name="label_authorize_sign_txn">Combined authorize and sign txn x1</string>
+    <string name="label_authorize_sign_txn">Authorize + Txn</string>
```

Comment:
NIT: should this be "Authorize + Sign Txn" or is that too big for the available space?

Reply:
Yeah, it was running right up against the available space. That screen is getting pretty crowded!